iosApp/Podfile

```diff
@@ -6,6 +6,6 @@ target 'iosApp' do
   # use_frameworks!
 
   # Pods for iosApp
-  pod 'PurchasesHybridCommon'
-  pod 'PurchasesHybridCommonUI'
+  pod 'PurchasesHybridCommon', '11.1.1'
+  pod 'PurchasesHybridCommonUI', '11.1.1'
```

Comment:
Do we need to add these to the update PHC fastlane command: https://github.com/RevenueCat/purchases-kmp/blob/d85257604bc8ae57b33f0df559d450c9491ebb04/fastlane/Fastfile#L79

Reply:
Yesss we should, thanks! I thought we'd need to make a change to the PHC repo, but it's right here of course. Done: https://github.com/RevenueCat/purchases-kmp/pull/129/commits/ff83102fa017026ed60ac636c763e049be28e928